src/letsRoute.js

```diff
@@ -36,6 +55,18 @@ Anumargak.prototype._on = function (method, url, fn) {
         }
     }
 
+    var namedExpressionMatches = getNamedExpressionMatches(url, namedExpressionRegexStr);
```

Comment:
We can extract this logic to a function.

Reply:
I didn't get. Do you mean to say move to utils?

Reply:
No, just extract the logic to another function in the same class. Eg `function replaceNamedExpressions()`. So that the size of `Anumargak.prototype._on()` will not increase.